Sources/ProjectDescription/SettingsTransformers.swift

```diff
@@ -0,0 +1,137 @@
+    func codeSignIdentityAppleDevelopment() -> SettingsDictionary {
+        codeSignIdentity("Apple Development")
+    }
+
+    /// Sets "CODE_SIGN_IDENTITY" to `identity`
+    func codeSignIdentity(_ identity: String) -> SettingsDictionary {
+        merging(["CODE_SIGN_IDENTITY": SettingValue(identity)])
+    }
+
+    // MARK: - Versioning and Product Name
+
+    /// Sets "PRODUCT_NAME" to `name`
+    func productName(_ name: String) -> SettingsDictionary {
+        merging(["PRODUCT_NAME": SettingValue(name)])
+    }
+
+    /// Sets "CURRENT_PROJECT_VERSION" to `version`
+    func currentProjectVersion(_ version: String) -> SettingsDictionary {
+        merging(["CURRENT_PROJECT_VERSION": SettingValue(version)])
+    }
+
+    /// Sets "VERSIONING_SYSTEM" to "apple-generic"
+    func appleGenericVersioningSystem() -> SettingsDictionary {
+        merging(["VERSIONING_SYSTEM": "apple-generic"])
+    }
+
+    /// Sets "VERSION_INFO_PREFIX" to `version`. If prefix is not `nil`, it's used as "VERSION_INFO_PREFIX"; and suffix as "VERSION_INFO_SUFFIX"
+    func versionInfo(_ version: String, prefix: String? = nil, suffix: String? = nil) -> SettingsDictionary {
+        var versionSettings: SettingsDictionary = ["VERSION_INFO_STRING": SettingValue(version)]
+        versionSettings["VERSION_INFO_PREFIX"] = prefix.map { SettingValue($0) }
+        versionSettings["VERSION_INFO_SUFFIX"] = suffix.map { SettingValue($0) }
+
+        return merging(versionSettings)
+    }
+
+    // MARK: - Swift Settings
+
+    /// Sets "SWIFT_VERSION" to `version`
+    func swiftVersion(_ version: String) -> SettingsDictionary {
+        merging(["SWIFT_VERSION": SettingValue(version)])
+    }
+
+    /// Sets "OTHER_SWIFT_FLAGS" to `flags`
+    func otherSwiftFlags(_ flags: String...) -> SettingsDictionary {
+        merging(["OTHER_SWIFT_FLAGS": SettingValue(flags.joined(separator: " "))])
+    }
+
+    /// Sets "SWIFT_COMPILATION_MODE" to the available `SwiftCompilationMode` ("singlefile" or "wholemodule")
+    func swiftCompilationMode(_ mode: SwiftCompilationMode) -> SettingsDictionary {
+        merging(["SWIFT_COMPILATION_MODE": SettingValue(mode)])
+    }
+
+    /// Sets "SWIFT_OPTIMIZATION_LEVEL" to the available `SwiftOptimizationLevel` ("-O", "-Onone" or "-Osize")
+    func swiftOptimizationLevel(_ level: SwiftOptimizationLevel) -> SettingsDictionary {
+        merging(["SWIFT_OPTIMIZATION_LEVEL": SettingValue(level)])
+    }
+
+    // MARK: - Bitcode
+
+    /// Sets "ENABLE_BITCODE" to "YES" or "NO"
+    func bitcodeEnabled(_ enabled: Bool) -> SettingsDictionary {
+        merging(["ENABLE_BITCODE": SettingValue(enabled)])
+    }
+
+    // MARK: - Catalyst
+
+    /// Sets "DERIVE_MACCATALYST_PRODUCT_BUNDLE_IDENTIFIER" to "YES" or "NO"
+    func deriveMacCatalystProductBundleId(_ enabled: Bool) -> SettingsDictionary {
+        merging(["DERIVE_MACCATALYST_PRODUCT_BUNDLE_IDENTIFIER": SettingValue(enabled)])
+    }
```

Comment:
Ah my bad - both `SUPPORTS_MACCATALYST` and  `DERIVE_MACCATALYST_PRODUCT_BUNDLE_IDENTIFIER` are automatically set by Tuist.

https://github.com/tuist/tuist/blob/master/Sources/TuistGenerator/Generator/ConfigGenerator.swift#L243